src/iGenVar.cpp

```diff
@@ -35,6 +36,11 @@ void initialize_argument_parser(seqan3::argument_parser & parser, cmd_arguments
                       "Input long read alignments in SAM or BAM format (PacBio, Oxford Nanopore, ...).",
                       seqan3::option_spec::standard,
                       seqan3::input_file_validator{{"sam", "bam"}} );
+    parser.add_option(args.genome_file_path,
+                      'g', "input_genome",
+                      "Input the reference genome in FASTA or FASTQ format.",
+                      seqan3::option_spec::standard,
+                      seqan3::input_file_validator{{"fasta", "fa", "fastq"}} );
```

Comment:
You could add all valid file extensions (we really need some kind of defaults in seqan3)
```suggestion
                      seqan3::input_file_validator{{"fasta", "fa", "fna", "ffn", "ffa", "frn", "fastq", "fq"}} );
```

Reply:
I have seen `fna` before, the others should be double checked, but I think this is a good idea.
I found for FASTA in the german wiki: `.fa`, `.mpfa`, `.fna`, `.fsa`, `.fasta` and in the english wiki: .`fasta`, `.fna`, `.ffn`, `.faa`, `.frn`, `.fa`.
I think we don't have to support all of them, we can expect the user to customize his file extension, but a few like `fna` and `fq` we could possibly add.

Reply:
Good point, thanks! 
I agree that it should be a constant in SeqAn for sequence files in general.